debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerConnectorConfig.java

```diff
@@ -268,6 +276,10 @@ public SnapshotMode getSnapshotMode() {
         return snapshotMode;
     }
 
+    public Predicate<ColumnId> getColumnFilter() {
```

Comment:
So eventually this should be a dedicated functional interface `ColumnFilter` (see DBZ-1093). But we can go with this for now and alter this later on in one pass over the entire code base.